src/Symfony/Bridge/Twig/Node/FormThemeNode.php

```diff
@@ -28,7 +28,7 @@ public function compile(Compiler $compiler)
     {
         $compiler
             ->addDebugInfo($this)
-            ->write('$this->env->getRuntime(\'Symfony\Bridge\Twig\Form\TwigRenderer\')->setTheme(')
+            ->write('$this->env->getRuntime(\'Symfony\Component\Form\FormRenderer\')->setTheme(')
```

Comment:
At least the BC break should be documented in the CHANGELOG & UPGRADE files I'd say.